### PR TITLE
Correct russian translation

### PR DESCRIPTION
**Pull request by alsoknownasdrew**
Correct some wrong wordings that seamed to be produced by an automated translator

### CODE REVIEW

File: languages/ru.json [hunk 1]
```diff
@@ -6,59 +6,59 @@
     "Cancel Response": "Отменить ответ",
     "Cannot contact hCaptcha. Check your connection and try again.": "Не удается связаться с hCaptcha. Проверьте ваше соединение и попробуйте еще раз.",
     "Check": "Проверить",
-    "Close": "близко",
-    "Close Modal": "Закрыть Модал",
+    "Close": "Закрыть",
+    "Close Modal": "Закрыть Модальное Окно",
     "Enter the two words in the image.": "Введите два слова на картинке.",
```
Comment (alexeysofin) [inline languages/ru.json:11]: Maybe better "Введите два слова с изображения."?

File: languages/ru.json [hunk 2]
```diff
@@ -6,59 +6,59 @@
...
+    "Our accessibility option may help.": "Наш доступный вариант может помочь.",
     "Please also try turning off your ad blocker.‍": "Также попробуйте отключить блокировщик рекламы.",
     "Please describe your issue.": "Пожалуйста, опишите вашу проблему.",
     "Please provide details and steps to reproduce.": "Пожалуйста, предоставьте детали и шаги для воспроизведения.",
-    "Please select an image to report.": "Пожалуйста, выберите изображение для отчета.",
+    "Please select an image to report.": "Пожалуйста, выберите изображение для жалобы.",
     "Please select an option to send response.": "Пожалуйста, выберите опцию, чтобы отправить ответ.",
     "Please try again.": "Пожалуйста, попробуйте еще раз.",
     "Privacy": "Конфиденциальность",
     "Provide feedback for hCaptcha.": "Предоставить отзыв на hCaptcha.",
```
Comment (alexeysofin) [inline languages/ru.json:40]: "Оставить отзыв об hCaptcha."?

File: languages/ru.json [hunk 3]
```diff
@@ -6,59 +6,59 @@
...
     "Report Bug": "Сообщить об ошибке",
-    "Report Image": "Сообщить об изображении",
+    "Report Image": "Пожаловаться на изображение",
     "Report a software bug or issue to hCaptcha.": "Сообщить об ошибке или проблеме программного обеспечения в hCaptcha.",
-    "Report an image to hCaptcha.": "Сообщить об изображении в hCaptcha.",
+    "Report an image to hCaptcha.": "Пожаловаться на изображение в hCaptcha.",
     "Reporting a functionality issue?": "Сообщить о проблеме с функциональностью?",
-    "See how to report issues with detailed logs.": "Посмотрите, как сообщить о проблемах с подробными журналами.",
-    "Send": "послать",
+    "See how to report issues with detailed logs.": "Посмотрите, как сообщить о проблемах с подробными логами.",
```
Comment (alexeysofin) [inline languages/ru.json:51]: the word "лог" is purely a neologism inside IT community, hcaptcha has a broader audience, so maybe "Узнайте, как подробнее сообщать о проблемах."

File: languages/ru.json [hunk 4]
```diff
@@ -6,59 +6,59 @@
...
+    "Send": "Отправить",
     "Send Response": "Отправить ответ",
     "Send Us Feedback": "Отправить нам отзыв",
-    "Sign up for accessibility access to bypass the hCaptcha challenge.": "Подпишитесь на доступность доступа, чтобы обойти вызов hCaptcha.",
-    "Skip": "Пропускать",
-    "Skip Challenge": "Skip Challenge",
+    "Sign up for accessibility access to bypass the hCaptcha challenge.": "Зарегистрируйтесь на доступность доступа, чтобы обойти задачу hCaptcha.",
```
Comment (alexeysofin) [inline languages/ru.json:55]: That russian phrase does not make any sense, in english there's a connotation with the word "accessibility", haven't seen any analogue in russian, so I would translate that as "Попробуйте более доступный вариант, если испытываете проблемы с прохождением капчи." which says "Try out more accessible option, if you are having problems passing the captcha"

File: languages/ru.json [hunk 5]
```diff
@@ -6,59 +6,59 @@
...
+    "Skip": "Пропустить",
+    "Skip Challenge": "Пропустить задачу",
     "Software Bug": "Програмное обеспечение",
```
Comment (alexeysofin) [inline languages/ru.json:58]: Ошибка в программном обеспечении

File: languages/ru.json [hunk 6]
```diff
@@ -72,8 +72,8 @@
     "hCaptcha Privacy Policy": "Политика конфиденциальности hCaptcha",
     "hCaptcha Terms of Service": "Условия использования hCaptcha",
     "hCaptcha checkbox. Select in order to trigger the challenge, or to bypass it if you have an accessibility cookie.": "Флажок hCaptcha. Выберите, чтобы вызвать запрос или обойти его, если у вас есть файл cookie доступности.",
```
Comment (alexeysofin) [inline languages/ru.json:74]: "вызвать запрос" -> "пройти задание"

File: languages/ru.json [hunk 7]
```diff
@@ -72,8 +72,8 @@
...
-    "hCaptcha checkbox. You are verified I am human.": "Флажок hCaptcha. Вы проверены, я человек.",
+    "hCaptcha checkbox. You are verified I am human.": "Флажок hCaptcha. Вы провереный Ячеловек.",
     "hCaptcha is a service that reduces bots and spam by asking simple questions. Please follow the instructions at the top of the screen for each challenge. For more info visit": "hCaptcha - это сервис, который уменьшает ботов и спам, задавая простые вопросы. Пожалуйста, следуйте инструкциям в верхней части экрана для каждого испытания. Для получения дополнительной информации посетите",
```
Comment (alexeysofin) [inline languages/ru.json:76]: "уменьшает ботов и спам" -> "уменьшает количество ботов и спама"

File: languages/ru.json [hunk 8]
```diff
@@ -6,59 +6,59 @@
...
     "Having a problem?": "Возникла проблема?",
     "I am human": "Я человек",
     "If there are None, click Skip": "Если их нет, нажмите Пропустить.",
-    "Inappropriate": "несоответствующий",
+    "Inappropriate": "Неприемлемо",
     "Info": "Информация",
     "Information": "Информация",
     "Information about the hCaptcha service.": "Информация об услуге hCaptcha.",
-    "Learn about accessibility options.": "Узнайте о вариантах доступности.",
+    "Learn about accessibility options.": "Узнайте о опциях для доступности.",
```
Comment (alexeysofin) [inline languages/ru.json:26]: "Узнайте о более доступных способах прохождения капчи."? Again just the word accessibility may be misunderstood without icon or some other context

File: languages/ru.json [hunk 9]
```diff
@@ -6,59 +6,59 @@
...
-    "Close": "близко",
-    "Close Modal": "Закрыть Модал",
+    "Close": "Закрыть",
+    "Close Modal": "Закрыть Модальное Окно",
     "Enter the two words in the image.": "Введите два слова на картинке.",
     "Error fetching captcha.": "Ошибка при получении капчи.",
     "Error submitting captcha.": "Ошибка при отправке капчи.",
     "Feedback": "Обратная связь",
-    "Get information about hCaptcha and accessibility options.": "Получите информацию о hCaptcha и опциях доступности.",
+    "Get information about hCaptcha and accessibility options.": "Получите информацию о hCaptcha и опциях для доступности.",
```
Comment (alexeysofin) [inline languages/ru.json:15]: "Узнайте о более доступных способах прохождения капчи."

**Comment by alsoknownasdrew** [comment]
Thanks for all the feedback @alexeysofin I'm going to update this PR this afternoon 👍

**Comment by alexeysofin** [comment]
@alsoknownasdrew thanks :)

**Comment by e271828-** [comment]
@alexeysofin if you approve please leave a review :)

**Review by alexeysofin — APPROVED** [review]

**Comment by alexeysofin** [comment]
@e271828- approved, but honestly, overall russian hcaptcha website looks bad, some core words like "image annotation" are translated incorrectly, which might break some natural SEO traffic, we might need to take a look there

**Pull request merged**